client/src/home/sidebar/Sidebar.js

```diff
@@ -0,0 +1,78 @@
+import React, {useState, useEffect} from 'react';
+
+import {useCookies} from 'react-cookie';
+import {getAllItems} from '../../MongodbFunctions';
+import {Form} from 'react-bootstrap';
+
+import {SidebarItemAlpha, SidebarItemExpiry} from './SidebarItem';
+
+import './Sidebar.scss';
+
+function Sidebar() {
+  const [itemsAlpha, setItemsAlpha] = useState([]);
+  const [itemsExpiry, setItemsExpiry] = useState([]);
+  const [cookies] = useCookies(['token']);
+  const [sortBy, setSortBy] = useState('Alphabetical');
+  const [expandAll, setExpandAll] = useState(false);
+
+  const sortAlpha = (a, b) => {
+    return a.name > b.name;
+  }
+
+  const sortExpiry = (a, b) => {
+    a = new Date(a.outDate);
+    b = new Date(b.outDate);
+    let diff = a.getTime() - b.getTime();
+    if (diff != 0) {
```

Comment:
should this be a `!==` ?
```suggestion
    if (diff !== 0) {
```